From d38372464ba79c6729d5c4ed90154317f02e9479 Mon Sep 17 00:00:00 2001
From: David S. Wang <dsw@cloudera.com>
Date: Wed, 27 Mar 2013 16:40:54 -0700
Subject: [PATCH 41/96] HBASE-6972 HBase Shell deleteall should not require column to be defined (addendum)

Reason: Bug
Author: Ricky Saltzer
Ref: CDH-10398
(cherry picked from commit 41298d0057a21be7e0f22787c7c08aa09795286e)
---
 src/main/ruby/shell/commands/deleteall.rb |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/src/main/ruby/shell/commands/deleteall.rb b/src/main/ruby/shell/commands/deleteall.rb
index 0f71744..69d8efa 100644
--- a/src/main/ruby/shell/commands/deleteall.rb
+++ b/src/main/ruby/shell/commands/deleteall.rb
@@ -39,11 +39,13 @@ t to table 't1', the corresponding command would be:
 EOF
       end
 
-      def command(table, row, column, timestamp = org.apache.hadoop.hbase.HConstants::LATEST_TIMESTAMP)
+      def command(table, row, column = nil,
+                  timestamp = org.apache.hadoop.hbase.HConstants::LATEST_TIMESTAMP)
         deleteall(table(table), row, column, timestamp)
       end
 
-      def deleteall(table, row, column = nil, timestamp = org.apache.hadoop.hbase.HConstants::LATEST_TIMESTAMP)
+      def deleteall(table, row, column = nil,
+                    timestamp = org.apache.hadoop.hbase.HConstants::LATEST_TIMESTAMP)
         format_simple_command do
           table._deleteall_internal(row, column, timestamp)
         end
-- 
1.7.0.4

